Add editor config (#1364)
CiviWiki/OpenCiviWiki: CONTRIBUTING.md

Compatible Versioning ("ComVer") is SemVer where every PATCH number is 0 (zero). This way, ComVer is backwards compatible with SemVer.

A ComVer release from 3.6 to 3.7 is just a SemVer release from 3.6.0 to 3.7.0. In other words, ComVer is safe to adopt since it is basically SemVer without ever issuing PATCH releases.

### Editor

> This project provides an `.editorconfig` file, with some style options such as indent

Reviewer: Remember punctuation, since this is documentation.
```suggestion
This project provides an `.editorconfig` file, with some style options such as indent.
```

#### VS Code

> To use the `.editorconfig` file in VS Code install the [`EditorConfig for VS Code`](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) extension. You can find more information [here]()

Reviewer: Let's add a bit of improved grammar and the missing link here.
```suggestion
To use the `.editorconfig` file in VS Code, install the [`EditorConfig for VS Code`](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) extension. See the [EditorConfig documentation](https://editorconfig.org/) for more information.
```

#### Pycharm

> To use the `.editorconfig` file in Pycharm enable the [`EditorConfig`](https://plugins.jetbrains.com/plugin/7294-editorconfig) plugin. You find more information [here](https://www.jetbrains.com/help/idea/editorconfig.html)

Reviewer: Minor grammar and rewording to match the VSCode section above.
```suggestion
To use the `.editorconfig` file in Pycharm, enable the [`EditorConfig`](https://plugins.jetbrains.com/plugin/7294-editorconfig) plugin. See the [IntelliJ IDEA EditorConfig documentation](https://www.jetbrains.com/help/idea/editorconfig.html) for more information.
```
